kong/dao/migrations/cassandra.lua

```diff
@@ -506,54 +515,43 @@ return {
         return err
       end
 
+      local upstreams = require("kong.dao.schemas.upstreams")
+      local default = upstreams.fields.healthchecks.default
+
       for _, row in ipairs(rows) do
-        if not row.hash_on or not row.hash_fallback then
-          row.hash_on = "none"
-          row.hash_fallback = "none"
---          row.created_at = nil
-          local _, err = dao.upstreams:update(row, { id = row.id })
+        if not row.healthchecks then
+
+          local _, err = dao.upstreams:update({
+            healthchecks = default,
+          }, { id = row.id })
           if err then
             return err
           end
         end
       end
     end,
-    down = function(_, _, dao) end  -- n.a. since the columns will be dropped
-  },
-  {
-    name = "2017-11-07-192000_upstream_healthchecks",
-    up = [[
-      ALTER TABLE upstreams ADD healthchecks text;
-    ]],
-    down = [[
-      ALTER TABLE upstreams DROP healthchecks;
-    ]]
+    down = function(_, _, dao) end
   },
   {
-    name = "2017-11-07-192100_upstream_healthchecks_2",
+    name = "2017-10-27-134100_consistent_hashing_2",
```

Comment:
if the names are lexically sorted, then this one ends up before the other one. The design was like that, but I don't know whether that still is the case.

It would be a nasty hurdle for upgrading 0.12.0 to 0.12.1 if we change the names of the migrations, as they will then be run again. Not sure what happens in that case.

Reply:
They are still run in the order they are defined:

https://github.com/Kong/kong/blob/master/kong/dao/factory.lua#L271

Reply:
yes, they are actually loaded here: https://github.com/Kong/kong/blob/master/kong/dao/factory.lua#L193

but not sorted, after loading. So all seems well.